autocrud-context.tsx

```diff
@@ -0,0 +1,7 @@
+import { createContext } from 'react';
+
+export interface AutoCrudContextType<TItem> {
+  refreshGrid: () => void;
+}
+
+export const AutoCrudContext = createContext<AutoCrudContextType<any> | undefined>(undefined);
```

Comment:
autocrud.tsx defines a context using `<AutoCrudContext.Provider>` which allows the components inside the provider to use the context, as `useContext(AutoCrudContext)`. It is a way from crud to provide a helper method to the delete button, as it is not possible to pass properties directly to it.

Reply:
and it cannot be inside autocrud.tsx because then you have circular imports

Reply:
uh but wait what, why is this in the PROJECT root..

Reply:
deleted

Reply:
What I'm asking is: Should it be placed at the root package? Since currently I see two `autocrud-context.tsx` files (with slightly different contents):
* `autocrud-context.tsx`
* `packages/ts/react-grid/src/autocrud-context.tsx`
Or maybe there is something wrong with GH right now that shows the same file from two commits in two different places by mistake.